uni/ulsp/file_handler.icn

```diff
@@ -767,12 +767,21 @@ class Context(uniAll, uri, line, lineNum, charNum, objectName, methodName, conte
    end
 
    initially(uniAll, uri, lineNum, charNum)
+      local uniFile, uniLine, uniFileIndex := 0
       self.uniAll := uniAll
       self.uri := uri
       self.lineNum := lineNum
       self.charNum := charNum
       line := getDesiredLine(lineNum)
       contextCase := findContext()
+      uniFile := ((\uniAll.uniFilez)[uri]) | fail
+      totalFileLines := 0
+      if \uniFile then {
+         while uniLine := (uniFile.Read(uniFileIndex +:= 1) || "\n") do
+         {
+            totalFileLines +:= 1
+         }
+      }
```

Comment:
`uniFileIndex`  and `totalFileLines` are the same counter. Also `uniLine` isn't used. Are we  just trying to count the number of lines in a file?